app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/MessageController.kt

```diff
@@ -235,7 +235,7 @@ class MessageController @Inject constructor(
 
         fun deleteMessage(context: Context, mailbox: Mailbox, message: Message, realm: MutableRealm) {
```

Comment:
Shouldn't it called deleteMessageBlocking ?

Reply:
True, looks like I overlooked that one, and it's also called by at least one suspend function directly…